src/agent/README.md

```diff
@@ -49,6 +49,11 @@ $ rustup target add "${arch}-unknown-linux-musl"
 $ sudo ln -s /usr/bin/g++ /bin/musl-g++
 ```
 
+ppc64le-only: Manually install protoc, e.g.
```

Comment:
Static checks complaining on:
```
INFO: Spell checking file 'src/agent/README.md'
WARNING: Word 'protoc': did you mean one of the following?: proton, protocol, proctor
ERROR: Spell check failed for file: 'src/agent/README.md'
ERROR: spell check failed, See https://github.com/kata-containers/documentation/blob/master/Documentation-Requirements.md#spelling for more information.
INFO: spell check failed for document src/agent/README.md
Error: Process completed with exit code 1.
```

I think that just adding `` to the protoc word should fix it.